plugins/greynoise/plugin.spec.yaml

```diff
@@ -177,13 +266,462 @@ types:
+      required: false
+    intention:
+      title: Tag Intention
+      type: string
+      description: Tag Intention
+      required: false
+    name:
+      title: Tag Name
+      type: string
+      description: Tag Name
+      required: false
+  ip_sim:
+    actor:
+      title: Actor
+      type: string
+      description: Actor
+      required: false
+    asn:
+      title: ASN
+      type: string
+      description: ASN
+      required: false
+    city:
+      title: City
+      type: string
+      description: City
+      required: false
+    classification:
+      title: Classification
+      type: string
+      description: Classification
+      required: false
+    country:
+      title: Country
+      type: string
+      description: Country
+      required: false
+    country_code:
+      title: Country Code
+      type: string
+      description: Country Code
+      required: false
+    first_seen:
+      title: First Seen
+      type: string
+      description: First Seen
+      required: false
+    ip:
+      title: IP Address
+      type: string
+      description: IP Address
+      required: false
+    last_seen:
+      title: Last Seen
+      type: string
+      description: Last Seen
+      required: false
+    organization:
+      title: Organization
+      type: string
+      description: Organization
+      required: false
+  similar_ip:
+    actor:
+      title: Actor
+      type: string
+      description: Actor
+      required: false
+    asn:
+      title: ASN
+      type: string
+      description: ASN
+      required: false
+    city:
+      title: City
+      type: string
+      description: City
+      required: false
+    classification:
+      title: Classification
+      type: string
+      description: Classification
+      required: false
+    country:
+      title: Country
+      type: string
+      description: Country
+      required: false
+    country_code:
+      title: Country Code
+      type: string
+      description: Country Code
+      required: false
+    features:
+      title: Features Matched
+      type: "[]string"
+      description: Features Matched
+      required: false
+    first_seen:
+      title: First Seen
+      type: string
+      description: First Seen
+      required: false
+    ip:
+      title: IP Address
+      type: string
+      description: IP Address
+      required: false
+    last_seen:
+      title: Last Seen
+      type: string
+      description: Last Seen
+      required: false
+    organization:
+      title: Organization
+      type: string
+      description: Organization
+      required: false
+    score:
+      title: Similarity Score
+      type: float
+      description: Similarity Score
+      required: false
 connection:
   credentials:
     title: GreyNoise API Key
     description: API key from GreyNoise Account
     type: credential_secret_key
     required: true
     example: abcdefghijklmnopqrstuvwxyz0123456789
+triggers:
+  greynoise_alert:
```

Comment:
Could we change the name of this trigger to be more reflective of what it does? 😃 
Something like: `query_ips` or `monitor_greynoise_ips`